### DISCUSSION TITLE
Transfer ossareh/asdf-lsd

### DISCUSSION THREAD

**Discussion by ossareh** [mise-plugins/.github#6] [Transfer Request]
lsd is a ls replacement, you can read more about it here: https://github.com/lsd-rs/lsd

I use mise and I created ossareh/asdf-lsd last year so I can install lsd on my various hosts. I'd like to transfer it to mise-plugins.

**Comment by jdx** [comment]
sounds good, transfer it to my personal account and I'll put it in the org

**Reply by ossareh** [reply]
@jdx transfer initiated 👍 thanks!
